packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -1075,8 +1075,23 @@ export class IndexedDbLruDelegate implements ReferenceDelegate, LruDelegate {
     this.garbageCollector = new LruGarbageCollector(this);
   }
 
-  getTargetCount(txn: PersistenceTransaction): PersistencePromise<number> {
-    return this.db.getQueryCache().getQueryCount(txn);
+  getSequenceNumberCount(
+    txn: PersistenceTransaction
+  ): PersistencePromise<number> {
+    const docCountPromise = this.orphanedDocmentCount(txn);
+    const targetCountPromise = this.db.getQueryCache().getQueryCount(txn);
+    return targetCountPromise.next(targetCount =>
```

Comment:
We usually don't include `Promise` in the name. You can respond to this by just inlining the calls though :)

Reply:
I'm going to leave as-is. Inlining would put the operations in sequence, rather than in parallel. Also, the code is dealing with both the promise of the count, and the count itself, so I think it's worthwhile to distinguish them.